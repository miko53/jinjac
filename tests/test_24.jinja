{% if users and name %}
 -- condition 1 ok --
{% endif %}
{% if not users and not name %}
-- not possible  --
{% else %}
--> condition 2 ok --
{% endif %}
{% if name == @name %}
-- not possible 2 --
{% else %}
--> condition 3 ok -->
{% endif %}
{% if name != @name %}
--> condition 4 ok -->
{% endif %}
{% if gre == 547 or myNiceDouble < 0 %}
--> condition 5 ok -->
{% endif %}
{% if myNiceDouble < 0 or offset_array == 2 %}
--> condition 6 ok -->
{% endif %}
