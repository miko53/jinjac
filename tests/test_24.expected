
 -- condition 1 ok --


--> condition 2 ok --


--> condition 3 ok -->


--> condition 4 ok -->


--> condition 5 ok -->


--> condition 6 ok -->

